pkg/awsutils/awsutils.go

```diff
@@ -427,7 +440,7 @@ func New(useSubnetDiscovery, useCustomNetworking, disableLeakedENICleanup, v4Ena
 
 	// Clean up leaked ENIs in the background
 	if !disableLeakedENICleanup {
-		go wait.Forever(cache.cleanUpLeakedENIs, time.Hour)
+		go wait.Forever(func() { cache.cleanUpLeakedENIs(context.Background()) }, time.Hour)
```

Comment:
I think I may have confused you. When I asked if you can pass context around, I was trying to say if the context is managed and should be passed through the tree for better visibility. If they are all the same context like background(), I think it is ok to to get them when need them. This change looks like unnecessarily adding complexity.